engine/client/cl_main.c

```diff
@@ -2472,11 +2472,18 @@ void CL_ProcessFile( qboolean successfully_received, const char *filename )
 	{
 		if( filename[0] != '!' )
 			Con_Printf( "processing %s\n", filename );
+
+		if( Q_strlen( filename ) > 11 && Q_stristr( filename, "downloaded/" ) == filename )
```

Comment:
We really should at some point get rid of /downloaded folder and use `gamedir_download` like GoldSrc does...

Also, this is wrong, use `Q_strnicmp` if you want to compare the beginning of string.

Reply:
Ok replaced with `Q_strncmp`